DQMServices/FileIO/scripts/dqmMemoryStats.py

```diff
@@ -90,10 +90,10 @@ def displayDirectoryStatistics(stats, args):
     cutoff, display = args.cutoff * 1024, args.display
 
     as_list = [(v, k, ) for (k, v) in group_stats.items()]
-    as_list.sort(reverse=True, key=lambda (v, k): abs(v))
+    as_list.sort(reverse=True, key=lambda v_k1: abs(v_k1[0]))
```

Comment:
I think the old syntax is clearer here. Is there a problem with it ?

Reply:
I don't know. Will look into it.

Reply:
It is indeed clearer. I don't know why it does this replacement.

Reply:
here is the reason
https://www.python.org/dev/peps/pep-3113/

(I found this a bit more explanatory: https://stackoverflow.com/questions/15712210/python-3-2-lambda-syntax-error). 

and indeed the old syntax throws and error in python3

Python 3.6.4 (default, Apr 27 2018, 10:48:00) 
[GCC 6.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> dud=[(1,5),(2,3),(3,2),(4,20),(5,-1)]
>>> dud.sort(reverse=True,key=lambda (v,k): abs(v))
  File "<stdin>", line 1
    dud.sort(reverse=True,key=lambda (v,k): abs(v))
                                     ^
SyntaxError: invalid syntax